test/integration_test/fafb_matrix_test.go

```diff
@@ -0,0 +1,338 @@
+// +build integrationtest
+// +build fafb
```

Comment:
Why use both the tags and not just fafb?
Is the plan to run the whole operator test suite on fafb too?

Reply:
No particular reason, was just adding on to the existing tag with an AND in case there was some issue with having no tag or a different one. If it's okay to have a separate one I'll go ahead and change it.

Reply:
@piyush-nimbalkar actually one question is do we want to maybe have this build two different test binaries? We can then have flag in our test run script for which binary to run as well?

Reply:
I think a flag with single binaries is better.